src/mame/toaplan/toaplan2.cpp

```diff
@@ -416,10 +416,15 @@ constexpr unsigned toaplan2_state::T2PALETTE_LENGTH;
 ***************************************************************************/
 
 
-void toaplan2_state::machine_start()
+void ghox_state::machine_start()
 {
 	save_item(NAME(m_old_p1_paddle_h));
 	save_item(NAME(m_old_p2_paddle_h));
+}
+
+
+void truxton2_state::machine_start()
+{
 	save_item(NAME(m_z80_busreq));
 }
```

Comment:
It’s good practice to call the base `toaplan2_state::machine_start` even if it’s empty at the moment.  If for some reason it needs to be implemented in the future, it’s more convenient if the derived classes don’t all have to be changed at that point.